contracts/mocks/ERC1363PayableMock.sol

```diff
@@ -15,12 +15,12 @@ contract ERC1363PayableMock is ERC1363Payable {
     constructor(IERC1363 acceptedToken) ERC1363Payable(acceptedToken) {} // solhint-disable-line no-empty-blocks
 
     // solhint-disable-next-line no-unused-vars
-    function _transferReceived(address operator, address from, uint256 value, bytes memory data) internal override {
+    function _transferReceived(address /*operator*/, address /*from*/, uint256 /*value*/, bytes memory /*data*/) internal override {
         transferNumber = transferNumber.add(1);
     }
 
     // solhint-disable-next-line no-unused-vars
-    function _approvalReceived(address owner, uint256 value, bytes memory data) internal override {
+    function _approvalReceived(address /*owner*/, uint256 /*value*/, bytes memory /*data*/) internal override {
```

Comment:
Please remove instead of commenting and also remove the above solhint disable

```suggestion
    function _approvalReceived(address, uint256, bytes memory) internal override {
```